docs/self-managed/concepts/multi-region/dual-region.md

```diff
@@ -9,141 +9,125 @@ description: "A dual-region setup allows you to run Camunda in two regions synch
-
-#### Components
+We treat Elasticsearch as an **active-passive** component. While it can be stretched across regions, this is not officially supported and may impact Zeebe’s performance. Each region has its own Elasticsearch instance. Zeebe is exporting data to both instances and provides data replication.
 
-Camunda components are **active-passive** components and in their current state not highly available as we're limited by the included exporters that would cause data issues when running multiple instances at the same time.
+We recommend the approach of one Elasticsearch per region and configuring Zeebe to export the Elasticsearch data to both regions.
 
-In every region, there can be only a single component instance running (for example in Region 0, 1 instance of Operate, 1 instance of Tasklist; same in Region 1).
+#### Operate and Tasklist
 
-This means that one instance will be actively serving traffic, while the other is on standby.
+These components are **active-passive** components and, in their current state, not highly available as we're limited by the included exporters that would cause data issues when running multiple instances at the same time.
 
-##### Operate
+Only a single component instance can be running in every region (for example, in Region 0, there is 1 instance of Operate and 1 instance of Tasklist; the same is true in Region 1).
 
-In the event of a total active region loss, the following data will be lost:
+One instance must be actively serving traffic while the other is on standby.
 
-- Uncompleted batch operations
+Operate and Tasklist stores some data in memory. In the event of a total active region loss, the following data loss will occur:
 
-##### Tasklist
-
-In the event of a total active region loss, the following data will be lost:
-
-- Task assignments
+- Operate Losses: Uncompleted batch operations
+- Tasklist Losses: Task assignments
 
 ## Requirements and Limitations
 
 ### Installation Environment
 
-#### Kubernetes Setup
+Two Kubernetes clusters are required for the Helm chart installation.
 
-- Two Kubernetes clusters are required for the Helm chart installation.
-- OpenSearch (both managed and self-managed) is not supported in this dual-region setup.
+Amazon OpenSearch is **not supported** in dual-region configuration.
 
 #### Network Requirements
 
-- Kubernetes clusters, services, and pods must not have overlapping CIDRs. Each cluster must use distinct CIDRs that do not conflict or overlap with those of any other cluster; otherwise, you will encounter routing issues.
-- The regions (for example, two Kubernetes clusters) must be able to connect to each other (for example, via VPC peering). See [example implementation](/self-managed/setup/deploy/amazon/amazon-eks/dual-region.md) for AWS EKS.
+- Kubernetes clusters, services, and pods must not have overlapping CIDRs. Each cluster must use distinct CIDRs that do not conflict or overlap with those of any other cluster.
```

Comment:
We have reached the point where we have to mention the CIDR overlap explicitly, and now you're removing the reasoning behind its importance.

If I were a reader who had no idea about the topic, I would assume the limitation lies in the application.

I'm fine with the change, it's just funny how this topic has evolved.